sklearn/multioutput.py

```diff
@@ -121,7 +136,7 @@ def partial_fit(self, X, y, classes=None, sample_weight=None):
                 sample_weight, first_time) for i in range(y.shape[1]))
         return self
 
-    def fit(self, X, y, sample_weight=None):
+    def fit(self, X, y, sample_weight=None, fit_param=None):
```

Comment:
I would have done `**fit_params`

Reply:
+1 II think that we only define `fit_params` with `**fit_params`

Reply:
Thanks for your advice.  I will try to make the `fit_params` with  a more `sklearn` style.